mediacapture-streams/MediaStream-feature-policy.https.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<body>
+    <script src=/resources/testharness.js></script>
+    <script src=/resources/testharnessreport.js></script>
+    <script src=/common/get-host-info.sub.js></script>
+    <script>
+function withIFrame(url, allow)
+{
+    return new Promise(resolve => {
+        let frame = document.createElement('iframe');
+        frame.src = url;
+        frame.allow = allow;
+        frame.onload = () => { resolve(frame); };
+        document.body.appendChild(frame);
+    });
+}
+
+var crossDomain = get_host_info().HTTPS_REMOTE_ORIGIN;
+function runTestInIFrame(allow, crossOrigin)
+{
+    return new Promise(async resolve => {
+        window.addEventListener('message', function handler(event) {
+            resolve(event.data);
+            window.removeEventListener('message', handler);
+            iframe.remove();
+        });
+        let url = location.pathname + "#test";
+        if (crossOrigin)
+            url = crossDomain + url;
+        const iframe = await withIFrame(url, allow);
+    });
+}
+
+async function doTest()
+{
+    let results = { };
+    try {
+        await navigator.mediaDevices.getUserMedia({audio : true});
```

Comment:
Have you considered doing a `stream.getTracks().forEach(track => track.stop());` after each gUM call?

This might help on resource use, at least in the top-level domain, which I assume stays open while the iframes come and go, and perhaps help the test succeed on systems where hardware is used and is limited from being opened in multiple tabs at once maybe.

This might be related to why Firefox is failing. It may be running into [bug 1517555](https://bugzilla.mozilla.org/show_bug.cgi?id=1517555).